src/components/modals/EditDialogAdmin.tsx

```diff
@@ -376,23 +378,37 @@ class EditDialogAdmin extends React.Component<Props, ComponentState> {
 
         let renderData = this.getRenderData();
         return (
-            <div className={`cl-dialog-admin ${OF.FontClassNames.small}`}>
-                <div className="cl-ux-flexpanel">
-                    <div className="cl-ux-flexpanel--primary">
-                        <div className="cl-ux-flexpanel--left" style={{ width: "80%" }}>
-                            <div data-testid="traindialog-title" className={`cl-dialog-title cl-dialog-title--${editTypeClass} ${OF.FontClassNames.large}`}>
-                                <OF.Icon
-                                    iconName={isLogDialog ? 'UserFollowed' : 'EditContact'}
-                                />
-                                {isLogDialog ? 'Log Dialog' : 'Train Dialog'}
-                            </div>
+            <div className={`cl-dialog-admin`}>
+                <div className="cl-dialog-admin__header">
+                    <div data-testid="traindialog-title" className={`cl-dialog-title cl-dialog-title--${editTypeClass} ${OF.FontClassNames.xxLarge}`}>
+                        <OF.Icon
+                            iconName={isLogDialog ? 'UserFollowed' : 'EditContact'}
+                        />
+                        {isLogDialog ? 'Log Dialog' : 'Train Dialog'}
+                    </div>
+                    <div className={`cl-dialog-tags ${OF.FontClassNames.mediumPlus}`}>
+                        <div className="cl-field">
+                            <label htmlFor="tags">Tags:</label>
+                            <TagsInput
+                                id="tags"
+                                tags={this.props.tags}
+                                onAdd={this.props.onAddTag}
+                                onRemove={this.props.onRemoveTag}
+                            />
                         </div>
-                        <div className="cl-ux-flexpanel--right" style={{ width: "20%", marginRight: '3em' }}>
-                            <TrainingStatusContainer
-                                app={this.props.app}
+                        <div className="cl-field">
+                            <label htmlFor="description">Description:</label>
+                            <BorderlessInput
+                                id="description"
+                                placeholder="Click on to add description"
```

Comment:
Will need to internationalize at some point

Reply:
Yea, added internationalization for both components.